server/server/models/user.py

```diff
@@ -92,14 +91,30 @@ def __init__(self, email, nickname, password, create_date, location,
         self.avatar = avatar
         self.banned_to_date = banned_to_date
 
-
-def from_json(cls, data):
-    """ Deserializes User object """
-    return cls(**data)
+    @staticmethod
+    def read_user(request):
+        """ Method to read user from database """
+        user = User.get_one(request, id=request.matchdict['profile_id'])
+        if user:
+            dict_ = model_to_dict(user)
+        else:
+            raise exc.exception_response(404)
+        return dict_
+
+    @staticmethod
+    def update_user(json_data, request):
+        """ Method to update user in database """
+        if request.dbsession.query(User).get(request.matchdict['profile_id']):
+            request.dbsession.query(User)\
+                .filter(User.id == request.matchdict['profile_id']).\
+                update(json_data)
+        else:
+            raise exc.exception_response(404)
+        return {'status': 'Success!'}
 
 
-def model_to_json(sqlalchemy_object):
-    """ Serializes sqlalchemy_object to json
+def model_to_dict(sqlalchemy_object):
```

Comment:
You can use this function for all models, so it have no reason to store this function in user.py file. (In my opinion, will be better to save this function in __init__.py at model folder)